packages/model/addon/index.ts

```diff
@@ -39,4 +39,4 @@
   @public
  */
 
-export { Model as default, attr, belongsTo, hasMany } from './-private';
```

Comment:
We shouldn't re-export private-only things here

Reply:
Fixed